addAuthor.php

```diff
@@ -0,0 +1,80 @@
+<?php
+    include './dbconn.php';
+    //missing documentation
+    mb_internal_encoding("UTF-8");
+?>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" >
+    <link rel="stylesheet" href="./css/myquiz.css">
+    <title>Add Author</title>
+</head>
+<body>
+<?php
+    //That variable will be used to set the Value of the text field where AuthorNames are filled.
+    //Using this outside of POST puts blank name inside our input TEXT field for AuthorName
+    $wrongAuthor = '';
+    if ($_POST) {
+    //remove white spaces before and after filled AuthorName
+    $authorName = trim($_POST['authorName']);
+    //check if our Author exists in DB.
+    //If there's a match mysqli_num_rows becomes 1 because our Authors are Unique
+    //mysqli_num_rows returns 0 if there's no match.
+    $authorCheckQuery = "SELECT * FROM `authors` WHERE `name` = '$authorName'";
+    $authorCheckResult = mysqli_query($conn,$authorCheckQuery);
+    //this var will be used to check for Errors right before Inserting into DB
+    $errors = false;
+        if (
+            //missing documentation for mb_strlen
```

Comment:
Maybe TODO fits better here? The the IDE shows you a list of things you left for later

```suggestion
            // TODO: missing documentation for mb_strlen
```